### PR TITLE
fix: stories with i18n, removing provider

### PR DESCRIPTION
**Pull request by jsomsanith-tlnd**
**What is the problem this PR is trying to solve?**
There is an issue with react-i18next, described here : https://github.com/i18next/react-i18next/issues/750#issuecomment-508092824

Some stories use i18next provider, others don't, which introduce a bug, trying to destructure undefined.

**What is the chosen solution to this problem?**
Remove react-i18next provider from stories, and configure i18next, exposing the configuration.

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/stories/GuidedTour.js [hunk 1]
```diff
@@ -277,12 +277,10 @@ storiesOf('GuidedTour', module)
 	.addDecorator(story => (
 		<React.Fragment>
 			<LanguageSwitcher />
-			<I18nextProvider i18n={i18n}>
-				<React.Fragment>
-					{story()}
-					{getLayoutWithLoremIpsum()}
-				</React.Fragment>
-			</I18nextProvider>
+			<React.Fragment>
```
Comment (vbocquet) [inline packages/components/stories/GuidedTour.js]: No need to use this `<Fragment />` anymore?

File: packages/datagrid/stories/datagrid.component.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { IconsProvider } from '@talend/react-components';
 
+import './config/i18n';
```
Comment (vbocquet) [inline packages/datagrid/stories/datagrid.component.js]: Why you need to import this now? Don't understand, stories don't seem to be broke on datagrid

**Comment by vbocquet** [comment]
I just check all stories on surge, everything seem to be fix 👍

File: packages/datagrid/stories/datagrid.component.js (hunk 2 above)
Comment (annilsson) [reply packages/datagrid/stories/datagrid.component.js]: I think it's needed to make the `i18next.init()` code run, it's not exported but just run in the global scope so it's enough to do an import. Correct me if I'm wrong

**Pull request merged**